mobile-app/app/screens/AppNavigator/screens/Loans/VaultDetail/components/VaultDetailLoansRow.tsx

```diff
@@ -0,0 +1,549 @@
+
+  return (
+    <TouchableOpacity
+      style={tailwind("mb-2")}
+      onPress={() => {
+        navigation.navigate({
+          name: "BorrowMoreScreen",
+          merge: true,
+          params: {
+            vault: props.vault,
+            loanTokenAmount: props.loanToken,
+          },
+        });
+      }}
+    >
+      <ThemedViewV2
+        light={tailwind("bg-mono-light-v2-00")}
+        dark={tailwind("bg-mono-dark-v2-00")}
+        style={tailwind(
+          "py-4 px-5 flex flex-row items-center justify-between",
+          { "rounded-t-lg-v2": props.displaySymbol === "DUSD" },
+          { "rounded-lg-v2": props.displaySymbol !== "DUSD" }
+        )}
+      >
+        <View style={tailwind("flex flex-row items-center")}>
+          <SymbolIcon
+            symbol={props.displaySymbol}
+            styleHeight={36}
+            styleWidth={36}
+          />
+          {/* eslint-disable react-native/no-raw-text */}
+          <View style={tailwind("ml-2")}>
+            <ThemedTextV2
+              light={tailwind({
+                "text-gray-300":
+                  props.vaultState === LoanVaultState.IN_LIQUIDATION,
+                "text-mono-light-v2-900":
+                  props.vaultState !== LoanVaultState.IN_LIQUIDATION,
+              })}
+              dark={tailwind({
+                "text-gray-700":
+                  props.vaultState === LoanVaultState.IN_LIQUIDATION,
+                "text-mono-dark-v2-900":
+                  props.vaultState !== LoanVaultState.IN_LIQUIDATION,
+              })}
+              style={tailwind("text-sm font-semibold-v2")}
+              testID={`loan_card_${props.displaySymbol}`}
+            >
+              {`${new BigNumber(props.amount).toFixed(8)} ${
+                props.displaySymbol
+              }`}
+            </ThemedTextV2>
+            <ThemedTextV2
+              light={tailwind("text-mono-light-v2-700")}
+              dark={tailwind("text-mono-dark-v2-700")}
+              style={tailwind("text-xs font-normal-v2")}
+            >
+              {new BigNumber(props.interestAmount ?? 0).toFixed(8)}{" "}
+              {translate(
+                "screens/VaultDetailScreenLoansSection",
+                "as interest"
+              )}
+            </ThemedTextV2>
+          </View>
+        </View>
+
+        {props.vault !== undefined && (
+          <LoanActionButton
+            label={translate("components/VaultDetailScreenLoansSection", "Pay")}
+            disabled={!canUseOperations}
+            onPress={() => {
+              navigation.navigate({
+                name: "PaybackLoanScreen",
+                merge: true,
+                params: {
+                  vault: props.vault,
+                  loanTokenAmount: props.loanToken,
+                },
+              });
+            }}
+            testID={`pay_${props.displaySymbol}_loan`}
+          />
+        )}
+      </ThemedViewV2>
+      {props.vault !== undefined &&
```

Comment:
Should we add a condition to check if there's DUSD collateral?

Reply:
Added checking